mybids.php

```diff
@@ -14,7 +14,16 @@
     COALESCE(MAX(Bids.bidPrice), Auctions.startingPrice) AS highestBid,
     COUNT(Bids.bidID) AS bidCount,
     Auctions.endDate,
-    COALESCE(UserBids.userBidPrice, Auctions.startingPrice) AS userBidPrice
+    COALESCE(UserBids.userBidPrice, Auctions.startingPrice) AS userBidPrice,
+    CASE
+        WHEN NOW() < Auctions.endDate THEN 'Open'
+        WHEN NOW() >= Auctions.endDate 
+             AND COALESCE(UserBids.userBidPrice, 0) = COALESCE(MAX(Bids.bidPrice), 0) 
+             AND COALESCE(MAX(Bids.bidPrice), 0) >= Auctions.reservePrice THEN 'Won'

```

Comment:
Overall, it works well for me and looks great! However, I noticed that when we built the database, we didn't set a NOT NULL constraint on the reservePrice field. As a result, if an auction is created without a reservePrice, it will throw an error. We might need to consider whether to make the reservePrice field NOT NULL, or perhaps explore other options.

Reply:
great catch! I have updated the create table sql for auctions and made changes to handle any error that might be thrown when auction is created